auditlog/models/log.py

```diff
@@ -40,6 +40,11 @@ class AuditlogLog(models.Model):
         'auditlog.http.session', string=u"Session")
     http_request_id = fields.Many2one(
         'auditlog.http.request', string=u"HTTP Request")
+    type = fields.Selection(
```

Comment:
I'm asking myself: maybe I should change this name as it is a reserved keyword... s/type/log_type/